src/common/blocks/submit.ts

```diff
@@ -413,7 +413,7 @@ export const NewSubmissionBlock = (pullRequest: PullRequest, isUpdate: boolean =
   return [
     {
       color: getAttachmentColor(pullRequest.status),
-      fallback: `<@${pullRequest.author.id}> submitted a pull request  :rocket:`,
+      fallback: `*<@${pullRequest.author.id}> - <${pullRequest.link}|${_capitalizeString(pullRequest.type)} - ${_capitalizeString(pullRequest.title)}>*  :rocket:`,
```

Comment:
@YoungMayor sooo the 'fallback' field doesn't accept any form of formatting, the hyperlink for example will be ignored because it is used for notification and other things like the "Later" feature.

I would suggest this:

```suggestion
      fallback: `<@${pullRequest.author.id}> submitted a pull request  :rocket: \n*${_capitalizeString(pullRequest.type)} - ${_capitalizeString(pullRequest.title)}*`,
```

Reply:
<img width="485" alt="Screenshot 2024-07-03 at 12 45 34 PM" src="https://github.com/Ritcheyy/vickpr-bot/assets/29282112/a02f2815-dd17-4e59-ba94-82983fc42594">

this is how it will look like on the Later section

Reply:
Looks good